scripts/get_counts.py

```diff
@@ -411,6 +413,16 @@ def MergeDevData(dest_count_dir, ngram_order):
     log_file = dest_count_dir + '/log/merge_dev_counts.log'
     RunCommand(command, log_file, args.verbose=='true')
 
+# this function get the integer in max_memory formatted as '10G' or '1024'
+def ParseInt(string):
+    if string != '':
+      return int(''.join([x for x in string if x.isdigit()]))
+
+# this function get the arbitary string (letters or %) at the end of max_memory
+# For example, 'G', 'b', '%' in '10G', '10b', and '10%'
+def ParseChar(string):
+    if string != '':
+      return str(''.join([x for x in string if not x.isdigit()]))
```

Comment:
This two functions would not behave as the comment says. If the user passes a string like '1G0', it will be interpreted as 10G. Actually I think we need to check the input string for all scripts accepted --max-memory option. If the string is validated at beginning, here we can simply get the last char and check whether it is a digit, then do the divide.